{
  "modified": "2025-07-31T19:38:33Z",
  "published": "2025-07-31T19:38:33Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "GHSA-8ch6-ch56-9v9j",
    "MAL-2025-4868",
    "SNYK-JS-PYICEBERG-10338603"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "npm",
        "name": "pyiceberg"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-JS-PYICEBERG-10338603"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/npm/pyiceberg/MAL-2025-4868.json"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-8ch6-ch56-9v9j"
    },
    {
      "type": "PACKAGE",
      "url": "https://npmjs.com/package/pyiceberg"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "d8965e2298cf6df4f19a33465f346f412b9e28350315fed9dba273e9a2481141",
        "import_time": "2025-08-01T10:41:40.590649076Z",
        "id": "RLUA-2025-03995",
        "modified_time": "2025-07-31T19:38:33Z"
      }
    ]
  }
}
